<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTariffsR29</class>
 <widget class="QDialog" name="ImportTariffsR29">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Импорт тарифов для Архангельской области</string>
  </property>
  <widget class="QLabel" name="lblStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>398</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>5</y>
     <width>511</width>
     <height>302</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Загрузить из</string>
       </property>
       <property name="buddy">
        <cstring>edtFileName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileName"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectFile">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="chkFullLog">
       <property name="text">
        <string>Подробный отчет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUpdateTariff">
       <property name="text">
        <string>Обновлять совпадающие тарифы</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="CProgressBar" name="progressBar">
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="log"/>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Начать импорт</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtFileName</tabstop>
  <tabstop>btnSelectFile</tabstop>
  <tabstop>chkFullLog</tabstop>
  <tabstop>chkUpdateTariff</tabstop>
  <tabstop>log</tabstop>
  <tabstop>btnImport</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
